README.md

```diff
@@ -337,3 +337,53 @@ NAMESPACE_CLEANUP
 HTTP_STRESS_CONTAINER_IMAGE
 HTTP_SERVER_CONTAINER_IMAGE
 ```
+
+## Deployments per namespace cluster limits Workload
+
+The Deployments per namespace cluster limits workload playbook is `workloads/cluster-limits-deployments-per-ns.yml` and will run the Deployments per namespace cluster limits test on your RHCOS cluster.
```

Comment:
I think it would be better to have a cluster limits prefix for the limits workloads for them to be identified easily but It's just a thought, we can remove the prefix as well.